packages/app-mobile/contentScripts/markdownEditorBundle/useWebViewSetup.ts

```diff
@@ -57,7 +57,7 @@ const useWebViewSetup = ({
 	` : '';
 
 	const injectedJavaScript = useMemo(() => `
-		if (typeof markdownEditorBundle === 'undefined') {
+		if (typeof window.markdownEditorBundle === 'undefined') {
```

Comment:
The original check seems to have been broken because the `markdownEditorBundle` defines the `markdownEditorBundle` variable using a toplevel `var`. Due to `var` scoping[^1], the local `markdownEditorBundle` is always `undefined`, even when `window.markdownEditorBundle` exists.

See the [MDN documentation for `var`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#redeclarations). Additionally, note that 
```js
window.a = 5;
(() => {
    console.log(a);
    if (true) {
       var a = 6;
    }
    console.log(a);
})()
```
logs
```
undefined
6
```